templates/payments/payouts.html

```diff
@@ -22,10 +22,11 @@
             </table>
             {% if not suspended %}
             <form action="/payments/payout_group_create" method="post" id="submit_form_group_create">
+                <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
                 <input type="button" value="Create payout group" data-action="create a payout group" data-form="submit_form_group_create" data-toggle="modal" data-target="#confirm_submit" class="btn btn-primary submit_button" />
             </form>
             {% endif %}
         </div>
     </div>
 
-{% endblock %}
```

Comment:
remove irrelevant change (and similar instances in other html files)

Reply:
not sure what this means. i can see the endblock

Reply:
i think some editor has created that no newline at end of file thing. thats not my editor since im only using vi/vim and its the same editor ive been using since i got this laptop.

Reply:
readded the symbol.

Reply:
it has to be your editor, you have created this PR

Reply:
> it has to be your editor, you have created this PR

i dont think it is. i looked at the PR on lnwallet_beryllium:
[f65850b](https://github.com/zap-me/beryllium/commit/f65850ba40e547e04e8bf111c09d6e348025a89e)

and its there. not sure what the benefits are of having the "no new line at the end of file" symbol is but its definitely not my editor. 

Do we need that to be there? it only exists on the few pages and not all of them. If we dont need it, I think we should just delete it to keep it consistent across the other pages rather than having it in some places and not others.